apps/chat/src/components/Chat/ChatInput/ChatExternalControls.tsx

```diff
@@ -66,7 +67,9 @@ export default function ChatExternalControls({
           <span className="text-secondary">
             <IconCopy width={18} height={18} />
           </span>
-          {t('Duplicate the conversation to be able to edit it')}
+          {conversationsToDuplicate.length > 1
+            ? t('Duplicate conversations to be able to edit it')
```

Comment:
difference only with `s`
```suggestion
            ? t('Duplicate the conversations to be able to edit it')
```